src/components/Cells/Cells.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classes from './Cells.scss';
+
+const Cells = () => (
+    <section className={classes.Cells}>
+        <div className={classes.Cell}>a</div>
+        <div className={classes.Cell}>t</div>
+        <div className={classes.Cell}>l</div>
+        <div className={classes.Cell}></div>
+        <div className={classes.Cell}></div>
+        <div className={classes.Cell}></div>
+        <div className={classes.Cell}></div>
+    </section>
```

Comment:
1. fani semantycznego HTML-a postulowaliby zastosowanie `ul` i `li` tutaj :)

2. oczywiście docelowo generowanie linii 6-12 powinno się opierać na iterowaniu po zmiennej stanu (np. tablica siedmioelementowa, której elementy to litera lub undefined), taka zmienna mogłaby pochodzić ze reduxowego store'a, którego chcieliśmy poćwiczyć w projekcie :) to może być w oddzielnym pijarze

Reply:
Jak w komentarzu powyżej dot. `ul` i `li`